pages/cart/cart.css

```diff
@@ -0,0 +1,86 @@
+.cartpage h2{
+    margin:3rem 1rem; 
+    font-size:2.4rem;
+    text-align:center;
+}
+    .cart_products{
```

Comment:
Suggestion : Try adding a generic class( utility class) in your component library for commonly used flex property and use it in other projects.

Reply:
ok I will use generic class from next,  thanks  for the suggestion .